components/bl-dashlet-component/src/index.js

```diff
@@ -53,20 +60,26 @@ const useComponentActions = (component, size, setSize, position, setPosition, is
     setPosition: position => setPosition(position),
     getPosition: () => position,
     setIsOpen  : isOpen => setIsOpen(isOpen),
-    getIsOpen  : () => isOpen
+    getIsOpen  : () => isOpen,
   });
 };
 
-const useLocalSettings = (storage, position, size, isOpen) => {
+const useLocalSettings = (localStorage, storage, position, size, isOpen) => {
   useEffect(() => {
-    storage.position = position;
+    if (localStorage) {
+      storage.position = position;
+    }
   }, [position]);
 
   useEffect(() => {
-    storage.size = size;
+    if (localStorage) {
+      storage.size = size;
+    }
   }, [size]);
 
   useEffect(() => {
-    storage.isOpen = isOpen;
-  }, [isOpen]);
+    if (localStorage) {
+      storage.isOpen = isOpen;
+    }
+  }, [size, position, isOpen, localStorage]);
```

Comment:
Why does this hook have size, position, and localStorage deps, but the other has no?